rust/src/ipsec.rs

```diff
@@ -0,0 +1,165 @@
+extern crate libc;
+
+use std;
+use std::mem;
+use libc::c_char;
+
+use rparser::*;
+
+use ipsec_parser::*;
```

Comment:
Oh, I forgot to comment that. The ipsec-parser crate is of https://github.com/rusticata/ipsec-parser
It was supposed to be merged in a later commit